src/base/sql.js

```diff
@@ -1,29 +1,95 @@
 import fs from 'fs';
 import path from 'path';
 
-import pgp from 'pg-promise';
+import pgp from './pgp';
 
-export function load(basePath) {
+/**
+ * Create `QueryFile` objects out of all files in `basePath`. Generate an
+ * object containing those `QueryFile`s, keyed by the source SQL file's
+ * basename.
+ * @TODO: What happens here if any files in `basePath` are not valid SQL?
+ * @param {string} basePath
+ * @return {object}  keyed `QueryFile` objects
+ */
+export function load (basePath) {
   return fs.readdirSync(basePath).reduce((result, queryFile) => {
     const query = path.basename(queryFile, '.sql');
     result[query] = pgp.QueryFile(path.join(basePath, queryFile));
     return result;
   }, {});
 }
 
+/**
+ * Return SQL query to read all or a single record (by ID) from `table`
+ * @param {string} table
+ * @param [{Number|String}] Optional parseInt-able ID for read-single.
+ * @return {string}
+ */
+export function read (table, id) {
```

Comment:
...this is veering into query-builder territory though and it just feels like the wrong abstraction to me. That said, I'm open to leaving it and curious to see how it plays out for your upcoming project!